fastparquet/benchmarks/columns.py

```diff
@@ -1,13 +1,17 @@
-from contextlib import contextmanager
 import os
+import tempfile
+import time
+from contextlib import contextmanager
+
 import numpy as np
 import pandas as pd
-import time
+import shutil
 
-from fastparquet.util import join_path
+import sys
+from six import PY2
 
 from fastparquet import write, ParquetFile
-from dask.utils import tmpdir
```

Comment:
This doe snot appear to exist

Reply:
https://github.com/dask/dask/blob/master/dask/utils.py#L116 ?
It is reasonable to repeat this here, so that fastparquet does not depend on dask even when benchmarking. You'll understand that this module only really exists for developers' testing.